base/plugins/score-plugin-engine/Engine/Protocols/Joystick/JoystickSpecificSettings.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <QMetaType>
+#include <wobjectdefs.h>
+
+namespace Engine {
+
+    namespace Network {
+
+        struct JoystickSpecificSettings {
+            int32_t joystick_id;
+            int joystick_index;
```

Comment:
maybe int32_t for both ?

Reply:
Yes !

Reply:
Done. In fact i use int32_t to be sure that I can store the ID that SDL provide, but without including SDL headers directly in score.